Java/src/main/java/epam/Planes/experimentalPlane.java

```diff
@@ -0,0 +1,33 @@
+package epam.Planes;
+
+import epam.models.ClassificationLevel;
+
+
+public class experimentalPlane extends Plane {
+	private ClassificationLevel classificationLevel;
+
```

Comment:
Why did you remove the ExperimentalType variable?

Reply:
It was only used on constructor method, but it didn't have another action

Reply:
go it